src/guide/views/dropdown/DropdownExample.jsx

```diff
@@ -67,23 +68,62 @@ export default class DropdownExample extends Component {
 
         <Example>
           <Dropdown
+            labelProvider={this.dropdownLabelProvider}
+            onSelect={this.onSelectDropdownOption}
             options={this.dropdownItems}
+            optionLabelProvider={this.dropdownOptionLabelProvider}
             selectedOption={this.state.selectedDropdownOption}
-            onSelect={this.onSelectDropdownOption}
+          />
+        </Example>
+
+        <Example title="With readonly">
+          <Dropdown
+            isReadonly
             labelProvider={this.dropdownLabelProvider}
             optionLabelProvider={this.dropdownOptionLabelProvider}
+            onSelect={this.onSelectDropdownOption}
           />
         </Example>
 
-        <Example title="With readonly">
+        <Example title="Borderless">
           <Dropdown
+            isBorderless
             labelProvider={this.dropdownLabelProvider}
+            onSelect={this.onSelectDropdownOption}
+            options={this.dropdownItems}
             optionLabelProvider={this.dropdownOptionLabelProvider}
+            selectedOption={this.state.selectedDropdownOption}
+          />
+        </Example>
+
+        <Example title="Borderless with dot">
```

Comment:
May be an example of a dropdown with border and a dot. But  not sure if that's an intended use case ever.